### PR TITLE
add like model and sample_data

### PR DESCRIPTION
**Pull request by YoungSong99**

### CODE REVIEW

**Review by jrzricardo — APPROVED** [review]
solid code there bud

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#  index_likes_on_photo_id  (photo_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (fan_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Like < ApplicationRecord
+  belongs_to :fan, class_name: "User", counter_cache: true
+  belongs_to :photo, counter_cache: true
```
Comment (thierrychau) [inline app/models/like.rb:23]: Can you add a validation to make sure a user can only like the same photo once?

**Review by thierrychau — CHANGES_REQUESTED** [review]
Great job so far, I left a minor comment below. Also, I think it's missing a lot of the changes that should have been in this PR, looks like the other scaffold generations are not in here. Can you fix this? Thanks!

File: app/assets/stylesheets/application.css
Comment (thierrychau) [inline app/assets/stylesheets/application.css:1]: It's generally good practice to keep pull request focused on the task assigned to you.
This makes pull requests with clean diffs, and makes it easier for reviewer also! Also, in bigger project, someone else might be responsible for frontend. 

Can I ask you to remove this? Thanks!

File: app/controllers/photos_controller.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  end
+
+  # GET /photos/1/edit
+  def edit
+  end
+
+  # POST /photos or /photos.json
+  def create
+    @photo = Photo.new(photo_params)
+    @photo.owner_id = current_user.id
```
Comment (thierrychau) [inline app/controllers/photos_controller.rb:25]: Great job!

File: app/controllers/photos_controller.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
```
Comment (thierrychau) [inline app/controllers/photos_controller.rb:26]: Remove extra line

File: app/views/devise/confirmations/new.html.erb
Comment (thierrychau) [inline app/views/devise/confirmations/new.html.erb:1]: We want to keep this PR as clean as possible. Can you move these devise views outside of this PR?

**Review by thierrychau — CHANGES_REQUESTED** [review]
Great job, I see the extra effort you put in to go above and beyond. However, we want to keep pull request very tight and concise, since it's also about collaboration. Can you see the comments below? Thanks!

**Review by thierrychau — APPROVED** [review]
Great job, you are good to merge. Thanks for the edits and your understanding!

**Pull request merged**